### PR TITLE
BLOG-73 | How to develop Angular 2 applications easily with Docker and angular-cli

### PR DESCRIPTION
**Pull request by hoatle**

### CODE REVIEW

File: source/_posts/2016-09-22-how-to-develop-angular-2-applications-easily-with-docker-and-angular-cli.markdown [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+keywords: Angular 2, easily, Docker, angular-cli, docker-angular-cli
+published: true
+---
+{% img center /images/2016/09/22/angular-boilerplate.png angular-boilerplate %}
+
+Everyone who is new to Angular 2 usually has many difficulties to get started with it, even with the
+getting started article from the Angular 2 official documentation. That's the reason why
+[angular-cli](https://cli.angular.io/) is created to ease the pain. `angular-cli` is a great CLI
+tool, however, it takes a lot of time to get it working by installing nodejs and `angular-cli`.
+Luckily, we create [docker-angular-cli](https://github.com/teracyhq/docker-angular-cli) project
```
Comment (hoavu) [inline source/_posts/2016-09-22-how-to-develop-angular-2-applications-easily-with-docker-and-angular-cli.markdown]: we have created the...

File: source/_posts/2016-09-22-how-to-develop-angular-2-applications-easily-with-docker-and-angular-cli.markdown [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+to build a Docker image so that anyone could use it right away for any Angular 2 projects without the
+hassle of installing nodejs and `angular-cli`. This guide will help you get started in the most
+easily and convenient way. It's assumed that you don't need master Docker but only require that
```
Comment (hoavu) [inline source/_posts/2016-09-22-how-to-develop-angular-2-applications-easily-with-docker-and-angular-cli.markdown]: --> you don't need to master Docker

File: source/_posts/2016-09-22-how-to-develop-angular-2-applications-easily-with-docker-and-angular-cli.markdown [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+# ng init --name <project-name> --style scss
+```
+
+You should see the <project-name> yourself. By default, the style file is `css`. I recommend using
+`scss` extension for style files.
+
+2. Add Dockerfile and docker-compose.yml file
+---------------------------------------------
+
+Add Dockerfile and docker-compose.yml like I did with https://github.com/teracyhq/angular-boilerplate/commit/082a879c5df3fa0ae6ddc000d84512754b62a642
```
Comment (hoavu) [inline source/_posts/2016-09-22-how-to-develop-angular-2-applications-easily-with-docker-and-angular-cli.markdown]: like I did --> as I did with...

File: source/_posts/2016-09-22-how-to-develop-angular-2-applications-easily-with-docker-and-angular-cli.markdown [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+and then you should see something like:
+
+```
+Starting virtual X frame buffer: Xvfb.
+Executing command /bin/bash
+root@6d15153f5858:~# 
+```
+And now you can execute any commands you want.
+
+That's it. Now you should start developing Angular 2 applications more easily with `docker-compose`
```
Comment (hoavu) [inline source/_posts/2016-09-22-how-to-develop-angular-2-applications-easily-with-docker-and-angular-cli.markdown]: That's it--> that's all. Now you can start...

Comment (hoatle) [inline source/_posts/2016-09-22-how-to-develop-angular-2-applications-easily-with-docker-and-angular-cli.markdown]: I'll still keep "That's it".

**Pull request merged**